### PR TITLE
jira tickets from comment removed

### PR DESCRIPTION
**Pull request by sandeshtamanq**

### CODE REVIEW

**Comment by VainXploits** [comment]
That's great but who asked

File: src/java/com/twitter/search/common/relevance/entities/GeoObject.java [hunk 1]
```diff
@@ -169,11 +169,8 @@ public static boolean approxEquals(GeoObject a, GeoObject b) {
     if (Double.compare(a.radius, b.radius) != 0) {
       return false;
     }
-    if (a.source != b.source) {
-      return false;
-    }
-
-    return true;
+    return a.source != b.source
```
Comment (arledesma) [inline src/java/com/twitter/search/common/relevance/entities/GeoObject.java:172]: It should be not not, not just not. Not that it matters, but one not is not the same as not not. This is not to be confused with not not not. Alternatively, no not.

**Review by arledesma — COMMENTED** [review]
To not or not to not, that is the question.

File: src/java/com/twitter/search/earlybird/document/ThriftDocumentPreprocessor.java [hunk 2]
```diff
@@ -69,7 +69,7 @@ public static ThriftDocument preprocess(
           "nullcastFilterSet");
 
   /***
-   * See JIRA SEARCH-7329
+   * 
    */
```
Comment (Taz03) [inline src/java/com/twitter/search/earlybird/document/ThriftDocumentPreprocessor.java:73]: remove the entire comment

File: src/java/com/twitter/search/earlybird/common/EarlybirdRequestLogger.java [hunk 3]
```diff
@@ -55,7 +55,7 @@ protected enum RootOnlyExtraFields {
     DEBUG_MODE,
     CACHE_HIT,
     USER_AGENT,
-    // See JIRA APPSEC-2303 for IP addresses logging
+
```
Comment (Taz03) [inline src/java/com/twitter/search/earlybird/common/EarlybirdRequestLogger.java:58]: y an empty new line at the end of enum?

**Pull request closed without merging**